python/lsst/obs/lsstSim/makeLsstSimRawVisitInfo.py

```diff
@@ -67,6 +72,12 @@ def setArgDict(self, md, argDict):
             self.pascalFromMmHg(self.popFloat(md, "PRESS")),
             float("nan"),
         )
+        # phosim doesn't supply LST or HA, so we have to approximate it (until we get UT1; DM-8053)
+        az, alt = argDict["boresightAzAlt"]
+        dec = argDict['boresightRaDec'][1]
+        ha = math.asin(-math.sin(az)*math.cos(alt)/math.cos(dec))*radians
```

Comment:
Would this be aided by explicit handling at the pole? I hope the result is `era=nan` (if nothing better can be computed) but it's not obvious to me that this is what happens.